File: cakk-domain/src/main/java/com/cakk/domain/repository/jpa/CakeShopOperationJpaRepository.java
package com.cakk.domain.repository.jpa;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.entity.shop.CakeShopOperation;

public interface CakeShopOperationJpaRepository extends JpaRepository<CakeShopOperation, Long> {

	List<CakeShopOperation> findAllByCakeShopId(Long cakeShopId);
}

File: cakk-infrastructure/persistence-mysql/src/main/java/com/cakk/infrastructure/persistence/repository/jpa/CakeShopOperationJpaRepository.kt
package com.cakk.infrastructure.persistence.repository.jpa

import com.cakk.infrastructure.persistence.entity.shop.CakeShopOperationEntity
import org.springframework.data.jpa.repository.JpaRepository

interface CakeShopOperationJpaRepository : JpaRepository<CakeShopOperationEntity, Long> {

	fun findAllByCakeShopId(cakeShopId: Long): List<CakeShopOperationEntity>
}

